airlock/permissions.py

```diff
@@ -151,6 +151,23 @@ def user_can_add_file_to_request(
     return True
 
 
+def check_user_can_replace_file_in_request(
+    user: User, request: "ReleaseRequest", workspace: "Workspace", relpath: UrlPath
+):
+    check_user_can_edit_request(user, request)
+    policies.check_can_replace_file_in_request(workspace, relpath)
+
+
+def user_can_replace_file_in_request(
+    user: User, request: "ReleaseRequest", workspace: "Workspace", relpath: UrlPath
+):  # pragma: no cover; not currently used
+    try:
+        check_user_can_update_file_on_request(user, request, workspace, relpath)
```

Comment:
should be `check_user_can_replace_file_on_request`?

Reply:
yikes, yes, thanks for spotting!